classes/options-general.php

```diff
@@ -319,7 +319,7 @@ function print_page() {
 
 													<tr valign="top">
 														<td scope="row"><label
-																for="default_state"><?php _e( 'Default State/Province', 'simplemap' ); ?></label>
+																for="default_state"><?php _e( 'Default State/Province', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 145 characters

<!-- Comment by [SideCI](https://sideci.com) -->